docs/src/architecture/npm-debug.log

```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```

Comment:
Thanks for pointing this out. I've rebased and removed npm-debug, and added it to gitignore which is checked in.

btw, you just volunteered to review and merge this :D